public/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+  <div class="container">
+    <p id="scrum">
+      <!--JUST MOVE THIS TO ANOTHER PAGE WHEN NEEDED.-->
+      <h3>SCRUM Plan HTML page</h3>
+      <b>Sprint One</b> Friday (04/10 - Monday 07/10):
+      <ul>
+        <li>The API description must be almost (we are using an iterative process) complete and available as a page on
+          your deployed project.</li>
+        <li>The SCRUM plan for the three mini-sprints must be available as a page on your deployed project. This can
+          either be a copy of our suggestion below, or Sprint-1 as given below and your own plan for the remaining
+          sprints if your “product owner” agrees (red students).</li>
+        <li>The CI-pipeline must be setup</li>
+        <li>Some of the Entity Classes and the Facade(s) must be ready with supplementing tests</li>
+      </ul>
+      <b>Sprint Two</b> (Tuesday 8/10 - Wednesday 9/10):
+      <ul>
+        <li>Most of the Entity Classes should be ready</li>
+        <li>Sample data should be available in the dev-database</li>
+        <li>Some of the endpoints (as a minimum a GET, POST and PUT) must be ready with the corresponding DTO’s and
+          integrations tests</li>
+      </ul>
+      <b>Sprint Three</b> (Thursday 10/10- Sunday 13/10):
+      <ul>
+        <li>Complete the API (as much as you have time for)</li>
+        <li>Implement a simple SPA which as a minimum must have the ability to use some of your GET endpoints and at
+          least one POST endpoint. Consider pages like:</li>
+        <ul>
+          <li>Get all persons with a given hobby</li>
+          <li>Get all persons living in a given city (i.e. 2800 Lyngby)</li>
+          <li>Get the count of people with a given hobby</li>
+          <li>Get a list of all zip codes in Denmark</li>
+          <li>Get a list of companies with more than xx employes</li>
+          <li>Create a Person (with hobbies, phone, address etc.)</li>
+        </ul>
+        <li>Complete documentation and prepare for your review presentation after the holiday</li>
+      </ul>
+      <p>Each sprint must end with a (very) short combined sprint retrospective meeting/sprint planning meeting where
+        you must demonstrate what you have done/plan to do. This meeting will be online using Flow. 50% of all available
+        studypoints for this CA will be given after these meetings, so make sure you all attend, and that you all get
+        something to say as for what you have done.
+      </p>
+
+    </p>
```

Comment:
This is the actual work done. The rest are just the bundle.js stuff.

Reply:
So maybe we can just take this out, if this branch is otherwise shit, and insert this properly somewhere else. Because this is the essence of this Pull Request.